docs/source/membership/membership.md

```diff
@@ -1,43 +1,86 @@
-# Membership
+# Membership Service Provider (MSP)
 
-If you've read through the documentation on [identity](../identity/identity.html)
-you've seen how a PKI can provide verifiable identities through a chain
-of trust. Now let's see how these identities can be used to represent the
-trusted members of a blockchain network.
+## Why do I need an MSP?
 
-This is where a **Membership Service** Provider (MSP) comes into play ---
-**it identifies which Root CAs and Intermediate CAs are trusted to define
-the members of a trust domain, e.g., an organization**, either by listing the
+Because Fabric is a permissioned network, blockchain participants need a way to
+prove their identity to the rest of the network in order to transact on the
+network. If you've read through the documentation on [identity](../identity/identity.html)
+you've seen how a PKI can provide verifiable identities through a chain
+of trust. How is that chain of trust used by the blockahin network?
+Recall in the credit card scenario that the Certificate Authority is
+like a card provider — it dispenses many different types of verifiable
+identities. An MSP, on the other hand, is like the list of card providers
+accepted by the store, determining which identities are the trusted
+members (actors) of the store  payment network.
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove
+you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Or consider a consortium of banks that each operate peer
```

Comment:
I would rephrase the paragraph below to make it a bit easier to follow:

Or consider a consortium of banks that operate a blockchain network. Each bank would operate peer and ordering nodes, and endorse transactions submitted to the network. However, each bank would also have departments and account holders. The account holders would belong to each organization, but would not run nodes on the network. They would only interact with the system from their mobile or web application.

Reply:
Thanks. Done